pkg/daemon/drain.go

```diff
@@ -38,16 +45,19 @@ func (dn *Daemon) cordonOrUncordonNode(desired bool) error {
 		err := drain.RunCordonOrUncordon(dn.drainer, dn.node, desired)
 		if err != nil {
 			lastErr = err
-			glog.Infof("cordon/uncordon failed with: %v, retrying", err)
+			glog.Infof("%s failed with: %v, retrying", verb, err)
```

Comment:
@kikisdeliveryservice , any interest in this portion of the PR, or should we just close this?

Reply:
yep, it makes sense to keep this part.

Reply:
Rerolled to keep this, but leave eventing alone, in 5e2ca0f6 -> f302d7e4.  The new commit message tries to motivate logging but not including events within `cordonOrUncordonNode`, but let me know if my attempt at that motivation doesn't make sense.